src/main/java/athena/Athena.java

```diff
@@ -36,7 +39,13 @@ public void runProgram() {
 
         ui.printWelcomeMessage();
 
-        taskList = storage.loadTaskListData();
+        try {
+            taskList = storage.loadTaskListData();
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (ArrayIndexOutOfBoundsException e) {
+            ui.printInvalidTask();
+        }
```

Comment:
I don't think we should print the stack trace to the user at all (as done by `e.printStackTrace()`) no matter what happens. We should give an error message saying the program failed to load the file. Perhaps we could make our own exception like `StorageLoadFailedException` so that we can customize its behaviour too?

Actually, I realized `ui.printInvalidTask` is ambiguous to the user too. The user wouldn't know which task is disposed and this is not really a good thing. I think it would be good to change this in this PR too.

https://github.com/AY2021S1-CS2113T-W12-2/tp/blob/4d422985a6a9c8baadd0397b5c10a979bbb5267b/src/main/java/athena/Ui.java#L134-L139

Also it is not immediately obvious that `ArrayIndexOutOfBoundsException` occurs when the save file is corrupted. Perhaps we should catch this exception in `Storage` and throw a custom exception like `StorageCorruptedException`.